procmon/processmonitor.cpp

```diff
@@ -5039,48 +5014,6 @@ void ProcessMonitor::checkModuleFailover( std::string processName)
     }
 }
 
-/******************************************************************************************
-* @brief	runUpgrade
-*
-* purpose:	run upgrade script
-*
-******************************************************************************************/
-/*int ProcessMonitor::runUpgrade(std::string mysqlpw)
-{
-	Oam oam;
-
-	for ( int i = 0 ; i < 10 ; i++ )
-	{
-		//run upgrade script
-		string cmd = startup::StartUp::installDir() + "/bin/upgrade-columnstore.sh doupgrade --password=" +
-			mysqlpw + " --installdir=" +  startup::StartUp::installDir() + "  > " + "/tmp/upgrade-columnstore.log 2>&1";
-		system(cmd.c_str());
-
-		cmd = "/tmp/upgrade-columnstore.log";
-		if (oam.checkLogStatus(cmd, "OK")) {
-			log.writeLog(__LINE__, "upgrade-columnstore.sh: Successful return", LOG_TYPE_DEBUG);
-			return oam::API_SUCCESS;
-		}
-		else {
-			if (oam.checkLogStatus(cmd, "ERROR 1045") ) {
-				log.writeLog(__LINE__, "upgrade-columnstore.sh: Missing Password error, return success", LOG_TYPE_DEBUG);
-				return oam::API_SUCCESS;
-			}
-
-			log.writeLog(__LINE__, "upgrade-columnstore.sh: Error return, check log /tmp/upgrade-status.log", LOG_TYPE_ERROR);
-			//restart mysqld and retry
-			try {
-				oam.actionMysqlCalpont(MYSQL_RESTART);
-			}
-			catch(...)
-			{}
-			sleep(1);
-		}
-	}
-	return oam::API_FAILURE;
-}
-*/
-
```

Comment:
I assume we replaced this with something else, but I don't think I've come across it yet.  Safe to get rid of this?